docs/content/latest/manage/backup-restore/point-in-time-recovery.md

```diff
@@ -121,26 +121,19 @@ This feature is in active development. YSQL and YCQL support different features,
 
 ### YSQL limitations
 
-* Currently, you can only recover from the following YSQL operations:
+* The following list summarizes what's supported:
   * Data changes
   * CREATE and DROP TABLE
-  * ALTER TABLE - only ADD, DROP, and RENAME COLUMN
   * CREATE and DROP INDEX
+  * ALTER TABLE - See [Alter Commands list](https://github.com/yugabyte/yugabyte-db/issues/1124) to understand Alters that work and those that don't.
 
-* The following types of ALTER TABLE aren't supported currently:
-  * ALTER TABLE RENAME TABLE
-  * Other types of ALTER TABLE supported by YSQL but not mentioned above
+* For Sequences, restoring rows of a table containing SERIAL columns and/or other columns of sequence type work. However, restoring to a state before the sequence table was created/dropped doesn't work.
```

Comment:
We should link to a GH issue with far more details about this, but keep this section strictly about what does not work.